server/mailet/integration-testing/src/test/java/org/apache/james/mailets/SubAddressingTest.java

```diff
@@ -97,43 +97,106 @@ void tearDown() throws IOException {
     void subAddressedEmailShouldBeDeliveredInINBOXWhenSpecifiedFolderDoesNotExist(@TempDir File temporaryFolder) throws Exception {
         setup(temporaryFolder);
 
-        //do not create mailbox
+        // do not create mailbox
+        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("Mailbox not found");
 
         sendSubAddressedMail();
         awaitSubAddressedMail(MailboxConstants.INBOX);
     }
 
     @Test
-    void subAddressedEmailShouldBeDeliveredInINBOXWhenNoRights(@TempDir File temporaryFolder) throws Exception {
+    void subAddressedEmailShouldBeDeliveredInINBOXWhenNobodyHasRight(@TempDir File temporaryFolder) throws Exception {
         setup(temporaryFolder);
 
         // create mailbox
-        testIMAPClient.connect(LOCALHOST_IP, jamesServer.getProbe(ImapGuiceProbe.class).getImapPort())
-            .login(RECIPIENT, PASSWORD)
-            .create(TARGETED_MAILBOX);
+        testIMAPClient.sendCommand("CREATE " + TARGETED_MAILBOX);
+        assertThat(testIMAPClient.sendCommand("GETACL " + TARGETED_MAILBOX)).contains("OK GETACL");
 
-        //do not give posting rights
+        // do not give posting rights
+        testIMAPClient.sendCommand("SETACL " + TARGETED_MAILBOX + " " + "anybody" + " -p");
```

Comment:
The RFC says `anyone` and not `anybody`. We shall be spec compliant! Likely the IMAP parsing layers shall be revised.